flash-messages/app-flash-messages.rb

```diff
@@ -0,0 +1,44 @@
+require "sinatra"
+
+# We point DataMapper to the correct database in setup.rb
+require "./setup"
+
+# models.rb is where we define our Chirp model
+require "./models"
+
+# To use flash messages, we need to require the Flash middleware
+require "rack-flash"
+
+# Because flash middleware uses sessions, we need to enable sessions
+set(:sessions, true)
+
+# Make sure to use a session secret!
+set(:session_secret, ENV["SESSION_SECRET"])
```

Comment:
Is this worth the conflation that may be happening? (I think so, after all it's good to have this kind of thing)